Actors/Actor.cs

```diff
@@ -1,21 +1,27 @@
 ﻿using Microsoft.Xna.Framework;
 using Microsoft.Xna.Framework.Graphics;
+using System.Collections.Generic;
+using TurnBasedFeest.Actions;
+using TurnBasedFeest.Actors.Behaviours;
+using TurnBasedFeest.Utilities;
 
 namespace TurnBasedFeest.Actors
 {
     class Actor
     {
         public string name;
-        private Vector2 position;
+        public Vector2 position;
         public Health health;
-        public bool moveRemaining;
+        public List<IAction> actions;
+        public ITurnBehaviour turnBehaviour; 
 
-        public Actor(string name, Vector2 position, int maxHealth, GraphicsDevice device)
+        public Actor(string name, Vector2 position, int maxHealth, List<IAction> actions, GraphicsDevice device, ITurnBehaviour turnBehaviour)
```

Comment:
GraphicsDevice is not used, nor should actors need this to build their own texture.